kie-wb-tests/kie-wb-tests-gui/src/main/java/org/kie/wb/selenium/model/Persp.java

```diff
@@ -21,156 +21,103 @@
-            = new Persp<>("Process Management",
-                          "Task Administration",
+            = new Persp<>("Manage",
+                          "Administration Tasks",
                           TaskAdministrationPerspective.class,
                           true);
-
+    public static final Persp<JobsPerspective> JOBS
+            = new Persp<>("Manage",
+                          "Jobs",
+                          JobsPerspective.class,
+                          true);
     public static final Persp<ExecutionErrorsPerspective> EXECUTION_ERRORS
-            = new Persp<>("Process Management",
+            = new Persp<>("Manage",
                           "Execution errors",
                           ExecutionErrorsPerspective.class,
                           true);
 
     public static final Persp<TasksPerspective> TASKS
-            = new Persp<>("N/A",
-                          "Tasks",
+            = new Persp<>("Track",
+                          "Tasks List",
                           TasksPerspective.class,
                           true);
-
     public static final Persp<ProcessAndTaskDashboardPerspective> PROCESS_AND_TASK_DASHBOARD
-            = new Persp<>("Dashboards",
-                          "Process & Task Dashboard",
+            = new Persp<>("Track",
+                          "Processes & Tasks",
                           ProcessAndTaskDashboardPerspective.class,
                           true);
-
-    public static final Persp<BusinessDashboardsPerspective> BUSINESS_DASHBOARDS
-            = new Persp<>("Dashboards",
+    public static final Persp<AppsPerspective> BUSINESS_DASHBOARDS
+            = new Persp<>("Track",
                           "Business Dashboards",
-                          BusinessDashboardsPerspective.class,
+                          AppsPerspective.class,
                           true);
 
-    public static final Persp<PluginManagementPerspective> PLUGIN_MANAGEMENT
-            = new Persp<>("Extensions",
-                          "Plugin Management",
-                          PluginManagementPerspective.class);
-
-    public static final Persp<AppsPerspective> APPS
-            = new Persp<>("Extensions",
-                          "Apps",
-                          AppsPerspective.class);
-
-    public static final Persp<DataSetsPerspective> DATA_SETS
-            = new Persp<>("Extensions",
-                          "Data Sets",
-                          DataSetsPerspective.class);
-
-    public static final Persp<DataSourcesPerspective> DATA_SOURCES
-            = new Persp<>("Extensions",
-                          "Data Sources",
-                          DataSourcesPerspective.class);
-
     private static final List<Persp<? extends AbstractPerspective>> ALL_PERSPECTIVES = Collections.unmodifiableList(Arrays.asList(
-            HOME_PAGE,
-            ADMIN,
-            TIMELINE,
-            PEOPLE,
             PROJECT_AUTHORING,
```

Comment:
This class used to represent the concept of "All perspectives accessible via top level navigation".
Are these all perspectives whose constants you're removing from here no longer accesible via MegaMenu? Could you please clarify / confirm the following?

1) I understand that ADMIN is now accessible via top-level link, no more separate menu item for it.

2) TIMELINE, PEOPLE and ADMINSTRATION are getting removed if I recall correctly.

3) What's happening to DATA_SETS, DATA_SOURCES, APPS, PLUGIN_MANAGEMENT, ARTIFACT_REPOSITORY which I see being removed from here?

4) What's this new DASHBOARDS constant? It seems to me from this PR that it points to the same thing as BUSINESS_DASHBOARDS..